compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -93,32 +40,43 @@ template <> class BCQFuser<1>
 public:
   bool fuseBCQ(loco::Graph *g)
   {
-    for (auto node : loco::all_nodes(g))
+
+    const auto output_nodes = loco::output_nodes(g);
+    for (auto node : output_nodes)
     {
-      if (auto circle_const = dynamic_cast<luci::CircleConst *>(node))
+      auto output_node = loco::must_cast<luci::CircleOutput *>(node);
+      if ((int)output_node->index() > _original_output_cnt)
```

Comment:
I'm not sure but `_orginal_output_cnt` might be variable that has the number of outputs without bcq related outputs. Then, isn't it `output_node->index() >= _original_output_cnt` that satisfy the condition as you intended? Because if the number of original output is 3, bcq node index that starts from 0 would starts from 3.

Reply:
You are understanding about the code very well but just one more thing, first output of BCQ model is metadata :)
Following example may help you
```
original_output_cnt = 2

BCQ_METADATA, original_output_1, original_output_2, BCQ_info1, BCQ_info2, ...
```

Reply:
@llFreetimell How about adding the comment as what you said?